File: samples/client/petstore/java-micronaut-client/src/test/java/org.openapitools/api/StoreApiTest.java
package org.openapitools.api;

import org.openapitools.model.Order;
import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Assertions;
import jakarta.inject.Inject;
import reactor.core.publisher.Mono;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;


/**
 * API tests for StoreApi
 */
@MicronautTest
public class StoreApiTest {

    @Inject
    StoreApi api;

    
    /**
     * Delete purchase order by ID
     *
     * For valid response try integer IDs with value &lt; 1000. Anything above 1000 or nonintegers will generate API errors
     */
    @Test
    public void deleteOrderTest() {
        String orderId = null;
        // api.deleteOrder(orderId).block();
        // Mono<Void> asyncResponse = api.deleteOrder(orderId);
        // TODO: test validations
    }

    
    /**
     * Returns pet inventories by status
     *
     * Returns a map of status codes to quantities
     */
    @Test
    public void getInventoryTest() {
        // Map<String, Integer> response = api.getInventory().block();
        // Mono<Map<String, Integer>> asyncResponse = api.getInventory();
        // TODO: test validations
    }

    
    /**
     * Find purchase order by ID
     *
     * For valid response try integer IDs with value &lt;&#x3D; 5 or &gt; 10. Other values will generated exceptions
     */
    @Test
    public void getOrderByIdTest() {
        Long orderId = null;
        // Order response = api.getOrderById(orderId).block();
        // Mono<Order> asyncResponse = api.getOrderById(orderId);
        // TODO: test validations
    }

    
    /**
     * Place an order for a pet
     */
    @Test
    public void placeOrderTest() {
        Order _body = null;
        // Order response = api.placeOrder(_body).block();
        // Mono<Order> asyncResponse = api.placeOrder(_body);
        // TODO: test validations
    }

    
}

File: samples/client/petstore/dart-jaguar/flutter_petstore/openapi/test/store_api_test.dart
import 'package:openapi/api.dart';
import 'package:test/test.dart';


/// tests for StoreApi
void main() {
  var instance = new StoreApi();

  group('tests for StoreApi', () {
    // Delete purchase order by ID
    //
    // For valid response try integer IDs with value < 1000. Anything above 1000 or nonintegers will generate API errors
    //
    //Future deleteOrder(String orderId) async 
    test('test deleteOrder', () async {
      // TODO
    });

    // Returns pet inventories by status
    //
    // Returns a map of status codes to quantities
    //
    //Future<Map<String, int>> getInventory() async 
    test('test getInventory', () async {
      // TODO
    });

    // Find purchase order by ID
    //
    // For valid response try integer IDs with value <= 5 or > 10. Other values will generated exceptions
    //
    //Future<Order> getOrderById(int orderId) async 
    test('test getOrderById', () async {
      // TODO
    });

    // Place an order for a pet
    //
    //Future<Order> placeOrder(Order body) async 
    test('test placeOrder', () async {
      // TODO
    });

  });
}
